source/plugins/wakatime/metadata.yml

```diff
@@ -79,3 +79,8 @@ inputs:
     description: Include other languages
     type: boolean
     default: no
+
+  plugin_wakatime_show_only_github_public_repos:
+    description: Show only github public repositories
+    type: boolean
+    default: no
```

Comment:
I think it'd be better to be consistant with similar options by naming it `plugin_wakatime_repositories_visibility` and be a toggle between `public` and `all` repositories, like what is currently done for actvitity plugin:
https://github.com/lowlighter/metrics/blob/5622b9fcc3b52ea3e68814cb8206f29f486ddea5/source/plugins/activity/metadata.yml#L42-L51
Makes it easier to add other filters in the future if needed (like archived, forks, etc.)